puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/plugins/foreman-tasks.yaml

```diff
@@ -4,9 +4,5 @@
     branch:
       - master:
           foreman_branch: develop
-      - 0.16.x:
-          foreman_branch: 1.23-stable
-      - 0.15.x:
-          foreman_branch: 1.22-stable
```

Comment:
@adamruzicka Should new versions get added here?

Reply:
That would make sense to me